src/components/AddPlaidBankAccount.js

```diff
@@ -177,9 +179,14 @@ class AddPlaidBankAccount extends React.Component {
                         {!_.isEmpty(this.props.text) && (
                             <Text style={[styles.mb5]}>{this.props.text}</Text>
                         )}
-                        {/* @TODO there are a bunch of logos to incorporate here to replace this name
-                        https://d2k5nsl2zxldvw.cloudfront.net/images/plaid/bg_plaidLogos_12@2x.png */}
-                        <Text style={[styles.mb5, styles.h1]}>{this.state.institution.name}</Text>
+                        <View style={[styles.flexRow, styles.alignItemsCenter, styles.mb5]}>
+                            <Icon
+                                src={getBankIcon(this.state.institution.name).icon}
+                                height={40}
+                                width={40}
```

Comment:
These hardcoded values should be put in the `variables.js` file.

Reply:
Updated!